File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
import java.util.*;
class Solution {
    public int solution(int[][] sizes) {
        int width = 0;
        int height = 0;
        for(int i=0; i<sizes.length; i++){
            int w = Math.max(sizes[i][0],sizes[i][1]);
            int h = Math.min(sizes[i][0], sizes[i][1]);
            width = Math.max(w,width);
            height = Math.max(h,height);
        }
        return width*height;
    }
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var answer = 0;
    let width = 0;
    let height = 0;
    for(let i=0; i<sizes.length; i++){
        let X = Math.max(sizes[i][0],sizes[i][1]);
        let Y = Math.min(sizes[i][0],sizes[i][1]);
        width = Math.max(width,X);
        height = Math.max(height,Y);
    }
    return width * height;
}